dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -81,13 +81,47 @@
       = t('user.delete_text_teacher_note')
     - data_hash = { confirm: t('user.delete_confirm') }
     = button_to t('user.delete'), registration_path(resource_name), data: data_hash, method: :delete, class: 'btn btn-danger'
+- if current_user.can_create_personal_login? && !current_user.under_13? && params[:showTakeYourAccountWithYou]
```

Comment:
Related to above `@user` comments, it seems `current_user` (correctly) is referenced here, not `@user`. 👍  So I don't think we need `@user` at all.